engine/include/cubos/engine/splitscreen/splitscreen.hpp

```diff
@@ -0,0 +1,19 @@
+/// @file
+/// @ingroup splitscreen-plugin
+
+#pragma once
+#include <cubos/core/reflection/external/glm.hpp>
+
+#include <cubos/engine/renderer/plugin.hpp>
+#include <cubos/engine/renderer/renderer.hpp>
+
+namespace cubos::engine
+{
+
+    /// @brief Splits the viewport recursively for the given cameras.
+    /// @param position Viewport position.
+    /// @param size Viewport size.
+    /// @param count How many cameras need to be fitted in to the given viewport.
+    /// @param viewport Output array where the viewports will be set.
+    void setViewportCameras(glm::ivec2 position, glm::ivec2 size, int count, glm::ivec2* positions, glm::ivec2* sizes);
```

Comment:
This doesn't need to be exposed outside the plugin right? Isn't it only used in the system of the plugin?